source/Cosmos.System2/Graphics/Bitmap.cs

```diff
@@ -518,5 +518,32 @@ public void Save(Stream stream, ImageFormat imageFormat)
             Array.Copy(imageData, 0, file, position, imageData.Length);
             stream.Write(file, 0, file.Length);
         }
+
+        /// <summary>
+        /// Resize a bitmap
+        /// </summary>
+        /// <param name="Width">New Bitmap Width.</param>
+        /// <param name="Height">New Bitmap Height.</param>
+        /// <param name="Image">The bitmap to resize.</param>
+        /// <returns>The resized bitmap.</returns>
+        public static Cosmos.System.Graphics.Bitmap ResizeBitmap(uint Width, uint Height, Cosmos.System.Graphics.Bitmap Image)
```

Comment:
We already have this method https://github.com/CosmosOS/Cosmos/blob/master/source/Cosmos.System2/Graphics/Canvas.cs#L894
I think its a valid use case to also have a public interface in the Bitmap class itself but we should not be implementing the same thing twice

Reply:
alr, I'll remove it